runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -295,6 +298,34 @@ pub trait API {
         -> Result<Vec<u8>, Error>;
 }
 
+/// Whitelisted EVM storge slots.
+///
+/// These slots are public and can be queried via `evm.Storage` queries.
+static WHITELISTED_MAGIC_SLOTS: Lazy<HashSet<H256>> = Lazy::new(|| {
+    // https://github.com/oasisprotocol/abi-playground/blob/49facdc2e49ccbbf4d2074662c920ee64fa16805/packages/nextjs/utils/abi-ninja/proxyContracts.ts#L4-L14
+    [
```

Comment:
@rube-de @matevz Please check this list if this list of slots is ok? (or are there any other slots we should include or if some of these are unneeded).

Reply:
I think `0xc5f16f0fcc639fa48a6947836d9850f504798523bf8c9a3a87d5876cf622bcf7` should also be added from the [EIP-1822](https://eips.ethereum.org/EIPS/eip-1822#functions).

Reply:
Then there are some other addresses listed by @lukaw3d 's abi playground fork https://github.com/oasisprotocol/abi-playground/blob/49facdc2e49ccbbf4d2074662c920ee64fa16805/packages/nextjs/utils/abi-ninja/proxyContracts.ts#L4-L14 and our docs https://docs.oasis.io/build/sapphire/develop/deployment, but those are mostly function address slots (ending with a number of `00000...0`). So I think we're good.

Reply:
> I think 0xc5f16f0fcc639fa48a6947836d9850f504798523bf8c9a3a87d5876cf622bcf7 should also be added from the [EIP-1822](https://eips.ethereum.org/EIPS/eip-1822#functions).

Had in in the list before, but @rube-de suggested to not support it, see the reasoning: https://github.com/oasisprotocol/oasis-sdk/pull/2215#discussion_r2093544706

> Then there are some other addresses listed by @lukaw3d 's abi playground fork

See @rube-de comments. Some of them are storage slots, and some of them are apparently not widely used/outdated, so it was suggested to not both supporting those.